lib/src/table/table.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, EventEmitter, Output, ViewEncapsulation } from "@angular/core";
+import { TableHeaderListItem, TableRowClickedEvent, SortInfo } from "./table.models";
+
+@Component({
+    selector: "seb-table",
+    templateUrl: "./table.component.html",
+    styleUrls: ["./table.component.scss"],
+    encapsulation: ViewEncapsulation.None,
+})
+export class TableComponent {
+    @Input() headerList: Array<TableHeaderListItem> = [];
+    @Input() sortInfo?: SortInfo;
+    @Input() selectable?: boolean = false;
+    @Input() isAllSelected?: boolean = false;
+    @Input() rows: Array<any> = [];
```

Comment:
can we make it generic instead of using `any`?

Reply:
same here, not sure if we can make this generic since the component willl be used in html using the selector. there is no way to pass the typescript model of the rows like we can in the table service.

Reply:
I will resolve this and keep the convo going in your original comment here https://github.com/sebgroup/ng-components/pull/293#discussion_r441447600